webapp/components/wpt-results.html

```diff
@@ -227,13 +233,23 @@
           this.refreshDisplayedNodes();
         };
 
+        // Add a diff testRun spoof, if needed.
+        if (this.diff && this.testRuns && this.testRuns.length == 2) {
+          this.fetchDiff(this.testRuns);
+        }
         // Fetch the results JSON for the TestRuns
+        await this.fetchResults(this.testRuns);
+        this.refreshDisplayedNodes();
+      }
+
+
+      async fetchResults(testRuns) {
         const testFileResults = await Promise.all(
-          this.testRuns.map(async json => this.fetchResults(json.results_url))
+          testRuns.map(async json => this.fetchResultsJSON(json.results_url))
```

Comment:
Nit: the `async` here is a bit odd since there's no `await`ing within. It does mean that it'll work even if `fetchResultsJSON` doesn't return a promise, but I still wonder if this isn't a typo/copypasta.

Reply:
I'll leave it alone for this PR (separate PR to fix welcome :)